proto/execution_stats.proto

```diff
@@ -106,6 +106,29 @@ message Execution {
   string command_snippet = 7;
 }
 
+message ExecutionNode {
+
+  // Remote execution node host
+  string host = 1;
+
+  // Remote execution node port
+  string port = 2;
+
+  // Assignable memory bytes in remote execution node
+  int64 assignable_memory_bytes = 3;
+
+  // Assignable cpu  in remote execution node
+  int64 assignable_milli_cpu = 4;
+
+  // Remote execution node operating system
+  string os = 5;
+
+  string arch = 6;
```

Comment:
nit: please add comments for the arch and pool fields.

Arch is the architecture of the remote execution node.
Pool is the remote execution pool that this node is assigned to.

Reply:
Done.